wgpu/src/util/init.rs

```diff
@@ -2,10 +2,10 @@ use wgt::{Backends, PowerPreference, RequestAdapterOptions};
 
 use crate::{Adapter, Instance, Surface};
 
-#[cfg(any(not(target_arch = "wasm32"), feature = "wgc"))]
+#[cfg(not(webgpu))]
```

Comment:
Rust isn't able to figure out that these appear regardless, probably because one is a typedef and the other a function.
![image](https://github.com/gfx-rs/wgpu/assets/1645124/9afe33da-402c-4443-8d0d-2a76e01b0116)
How about using `#[cfg_attr(docsrs, doc(cfg(all())))]` here?

Reply:
works for me if it does the trick 👍

Reply:
Done.